maximbircu/pull-request-assistant
# Do not block Friday command if controllers list is empty

Closes: #13 


### What has been done
Adjusted the code in a way so that Friday doesn't block commands in case the `controllers` list is empty.

### How to test
1. Configure Friday with an empty controllers config list for any repository;
2. Run any command and make sure Friday runs it.
